### PR TITLE
Add initial TravisCI integration

### PR DESCRIPTION
**Pull request by realityforge**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - echo "startup --output_base=$HOME/.cache/bazel" >> .bazelrc
+  # Avoid progress indicators that can blow out log which makes using WEB UI difficult
+  - echo "common --noshow_progress" >> .bazelrc
+  - echo "common --noshow_loading_progress" >> .bazelrc
+  # This is so we understand failures better
+  - echo "build --verbose_failures" >> .bazelrc
+
+script:
+  - bazel build :jre :jsinterop-annotations :jsinterop-annotations-j2cl :junit :minifier transpiler/java/... tools/java/...
+  - cd samples/helloworld/ && bazel build //...
```
Comment (jDramaix) [inline .travis.yml]: Could you call the new script I've just added: bazel_build_test.sh

Comment (realityforge) [reply .travis.yml]: Done.

Force pushed the new version.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+before_script:
+  - echo "startup --output_base=$HOME/.cache/bazel" >> .bazelrc
+  # Avoid progress indicators that can blow out log which makes using WEB UI difficult
+  - echo "common --noshow_progress" >> .bazelrc
+  - echo "common --noshow_loading_progress" >> .bazelrc
+  # This is so we understand failures better
+  - echo "build --verbose_failures" >> .bazelrc
+
+script:
+  - ./bazel_build_test.sh
```
Comment (jDramaix) [inline .travis.yml]: I dont think we need a list here. 
I can change that during the import process.

**Comment by realityforge** [comment]
Thanks!

**Comment by jDramaix** [comment]
Thank you to initiating that!

**Comment by realityforge** [comment]
It looks like the build failed :) Somehow it seems to be picking up an older version of bazel? Anyhoo - working on it locally to see if I can figure out what the issue is

**Comment by realityforge** [comment]
Okay - it seems like the failure is unrelated to bazel versions but due to lack of new line at the end of `.bazelrc`. Will send a patch to fix.

**Pull request closed without merging**